cascade.py

```
@@ -263,26 +783,78 @@ def get_event_nr(self):
+                        time.sleep(0.05)
+
+                    # Check if user force opens the door or cat is clean
+                    if self.bot.node_let_in_flag or (self.NO_PREY_FLAG and not self.PREY_FLAG):
+                        if self.use_surepet:
+                            self.open_catflap()
+                        else:
+                            logging.info("ℹ️  No backend available to open the catflap.")
+                            self.bot.send_text("ℹ️  No backend available to open the catflap.")
+                        self.reset_cumuli_et_al()
+
+                except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)